utils/index.ts

```diff
@@ -0,0 +1,42 @@
+import moment from 'moment';
+import {countries} from "@/assets/coutries"
+
+export function dateToHuman(date:Date|string):string|Date{
```

Comment:
the method is not appropriate `dateToHuman` -> `formatDate`

Reply:
resolved